english/utilize-wand/fork-rope-wand.md

```diff
@@ -3,6 +3,7 @@ you point your rope and fork wand to the door and firmly encant "Alohomora!"
 
     The door begins to acquire a dim yellow-green glow emanating from the lock mechanism.
     Rattling sounds out from the door's keyhole as the door begins to rumble within, wood
-    splintering as the sounds of grinding metal and wood fill the room.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.